### PR TITLE
feat(projects): Add link to create new team from project teams page

### PR DESCRIPTION
**Pull request by macqueen**
screenshot 2018-02-01 14 51 22

### CODE REVIEW

File: tests/js/spec/views/projectTeamSettings.spec.jsx [hunk 1]
```diff
@@ -78,10 +78,13 @@ describe('ProjectTeamsSettings', function() {
       expect(mock).not.toHaveBeenCalled();
 
       // open modal
-      wrapper.find('Button').simulate('click');
```
Comment (billyvg) [inline tests/js/spec/views/projectTeamSettings.spec.jsx]: curious why this doesn't work

**Review by billyvg — APPROVED** [review]
(We really need to change the "active" state on that dropdown button)

Would be nice to have some light dividers between teams, in the screenshot it looks like 1 team that's wrapped.

File: tests/js/spec/views/projectTeamSettings.spec.jsx (hunk 1 above)
Comment (macqueen) [reply tests/js/spec/views/projectTeamSettings.spec.jsx]: it was also finding the "create a new team" button from the dropdown and i was getting `Method “simulate” is only meant to be run on a single node. 2 found instead.`

**Comment by macqueen** [comment]
@billyvg 👍  i'm assuming we're gonna do a design walkthrough for these so i'll bring up the divider suggestion then. i just don't wanna make this inconsistent with other dropdowns

**Pull request merged**